src/DRICH_geo.cpp

```diff
@@ -37,90 +37,105 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
 
   // attributes, from compact file =============================================
```

Comment:
The majority of changes in this following block are `double` -> `auto` and formatting changes. There are some new variables though, for the PDU and sensor geometry.